packages/communication-react/CHANGELOG.beta.md

```diff
@@ -6,65 +6,58 @@ This log was last generated on Mon, 17 Jun 2024 16:27:12 GMT and should not be m
 
 ## [1.18.0-beta.1](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.18.0-beta.1)
 
-Mon, 17 Jun 2024 16:27:12 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.17.0...@azure/communication-react_v1.18.0-beta.1)
+Mon, 17 Jun 2024 16:27:12 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.17.0...1.18.0-beta.1)
 
 ### Features
-- GA CTE - Fix some stable build problems - FIx api changes from sdk ([PR #4530](https://github.com/azure/communication-ui-library/pull/4530) by jiangnanhello@live.com)
-- GA CTE - Fix some stable build problems - FIx api changes from sdk ([PR #4530](https://github.com/azure/communication-ui-library/pull/4530) by jiangnanhello@live.com)
-- Make AttachmentMetadata api public ([PR #4689](https://github.com/azure/communication-ui-library/pull/4689) by 77021369+jimchou-dev@users.noreply.github.com)
-- Version bumping communication-chat ([PR #4717](https://github.com/azure/communication-ui-library/pull/4717) by 9044372+JoshuaLai@users.noreply.github.com)
-- Soft Mute Participants in the call ([commit](https://github.com/azure/communication-ui-library/commit/not available) by edwardlee@microsoft.com)
-- Notification components to show notifications in the UI ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 96077406+carocao-msft@users.noreply.github.com)
-- Add 1:n call to CTE beta ([commit](https://github.com/azure/communication-ui-library/commit/not available) by jiangnanhello@live.com)
-- Add primary and secondary buttons to notification bar ([commit](https://github.com/azure/communication-ui-library/commit/not available) by 96077406+carocao-msft@users.noreply.github.com)
+
+- Data Loss Prevention support has been enabled in the Composites for Teams interoperability scenarios. When enabled in Teams, DLP policies can be applied to prevent sensitive information from being shared in chat messages.
+
+- UI library has been updated to include functionality for pinning a video tile from the participant list, in addition to the existing video gallery pinning feature.
```

Comment:
```suggestion
#### Policy Violation - Public Preview
- Data Loss Prevention support has been enabled in the Composites for Teams interoperability scenarios. When enabled in Teams, DLP policies can be applied to prevent sensitive information from being shared in chat messages.
#### Video Pinning - Public Preview
- UI library has been updated to include functionality for pinning a video tile from the participant list, in addition to the existing video gallery pinning feature.
```